rules/trimming.smk

```diff
@@ -30,6 +30,12 @@ rule trim_galore_SE:
         if [ "{params.fqsuffix}" != "fq" ]; then
           mv "$(dirname {output.se})/{wildcards.sample}_trimmed.fq.gz" {output.se}
         fi 
+        
+        # move the trimming report to the desired directory
+        report=$(dirname {output.se})/{wildcards.sample}.{params.fqsuffix}.gz_trimming_report.txt
+        if [[ -f $report ]]; then
+            mv $report {output.qc}
```

Comment:
If the report is not generated I want the rule to fail here, not that snakemake reports that the output is not generated (although this case should never happen.

Reply:
A similar system occurs elsewhere, I'll also remove it there then!